### PR TITLE
consolidate nix and add command for bundling in DNA version from acorn-hc releases

### PR DESCRIPTION
**Pull request by Connoropolous**
- [x] consolidate scripts and commands, into one nix file
- [x] update license to be CAL
- [x] add specific commands for bundling an acorn-hc DNA version, based on new strategy of uploaded DNA files to acorn-hc releases `acorn-bundle-dna`
- [x] add CAL LICENSE file

My rationale for going all in on nix is...
this repo is only really intended for those attempting to develop the acorn-release electron bundle for future packaging, and those trying to actually accomplish the packaging. Nix-shell is optimal in both cases, and edge cases don't seem concerning at the moment.

### CODE REVIEW

File: nix/acorn/default.nix [hunk 1]
```diff
@@ -1,41 +1,83 @@
...
 
- build = (pkgs.writeShellScriptBin "acorn-build" ''
+ build-linux = (pkgs.writeShellScriptBin "acorn-build-linux" ''
+  ${pkgs.nodejs}/bin/npm install
   set -euxo pipefail
   acorn_platform=''${1:-linux}
   acorn_arch=''${2:-x64}
-  ${update-deps}/bin/acorn-update-deps
   ${clean}/bin/acorn-clean
   electron-packager . Acorn --platform=$acorn_platform --arch=$acorn_arch --overwrite
```
Comment (samcooley) [inline nix/acorn/default.nix]: Propose adjusting this to sync with macOS build and hopefully get icon in:
` --prune=true --icon=\"ui/logo/acorn-logo-desktop-512px@2x.png\"`

File: nix/acorn/default.nix [hunk 2]
```diff
@@ -1,41 +1,83 @@
...
   chmod +x ./Acorn-$acorn_platform-$acorn_arch/Acorn
  '');
 
+ build-mac = (pkgs.writeShellScriptBin "acorn-build-mac" ''
```
Comment (samcooley) [inline nix/acorn/default.nix]: how about a copy of this as 'build-mac-unsigned' without the sign and notorize options?

Comment (Connoropolous) [reply nix/acorn/default.nix]: yeah... in favor

File: nix/acorn/default.nix (hunk 1 above)
Comment (Connoropolous) [reply nix/acorn/default.nix]: Setting the icon requires a different customization, what we have works for Mac only I *believe*

Comment (Connoropolous) [reply nix/acorn/default.nix]: But I will add prune

**Pull request merged**